src/ZoneServer/Skills/Handlers/Wugushi/WideMiasma.cs

```diff
@@ -0,0 +1,82 @@
+﻿using System;
+using System.Threading.Tasks;
+using Melia.Shared.L10N;
+using Melia.Shared.Game.Const;
+using Melia.Shared.World;
+using Melia.Zone.Network;
+using Melia.Zone.Skills.Handlers.Base;
+using Melia.Zone.Skills.SplashAreas;
+using Melia.Zone.World.Actors;
+using Melia.Zone.Skills.Combat;
+using System.Collections.Generic;
+using static Melia.Zone.Skills.SkillUseFunctions;
+
+namespace Melia.Zone.Skills.Handlers.Enchanter
+{
+	/// <summary>
+	/// Handler for the Wugushi skill Golden Frog.
+	/// </summary>
+	[SkillHandler(SkillId.Wugushi_WideMiasma)]
+	public class WideMiasma : IGroundSkillHandler
+	{
+		/// <summary>
+		/// Handles the skill, creates an area of effect that damages the enemies inside
+		/// </summary>
+		/// <param name="skill"></param>
+		/// <param name="caster"></param>
+		/// <param name="originPos"></param>
+		/// <param name="farPos"></param>
+		/// <param name="designatedTarget"></param>
+		public void Handle(Skill skill, ICombatEntity caster, Position originPos, Position farPos, ICombatEntity designatedTarget)
+		{
+			if (!caster.TrySpendSp(skill))
+			{
+				caster.ServerMessage(Localization.Get("Not enough SP."));
+				return;
+			}
+
+			skill.IncreaseOverheat();
+			caster.SetAttackState(true);
+			caster.StartBuff(BuffId.WideMiasma_Buff, skill.Level, 0, TimeSpan.FromSeconds(10), caster);
+
+			Send.ZC_SKILL_READY(caster, skill, caster.Position, caster.Position);
+			Send.ZC_NORMAL.UpdateSkillEffect(caster, caster.Handle, caster.Position, caster.Position.GetDirection(caster.Position), Position.Zero);
+			Send.ZC_SKILL_MELEE_GROUND(caster, skill, caster.Position, null);
+
+			// Radius seems precise
+			var radius = 150;
```

Comment:
It looks like the radius is 120 as per the skill_bytool.  It also looks like it's cut by half if it's in a PVP area.